payments-core/src/main/java/com/stripe/android/payments/bankaccount/ui/CollectBankAccountViewModel.kt

```diff
@@ -185,18 +190,30 @@ internal class CollectBankAccountViewModel @Inject constructor(
         result: FinancialConnectionsSheetInstantDebitsResult.Completed,
     ) {
         finishWithRefreshedIntent { intent ->
+            val paymentMethod = result.paymentMethod.parsePaymentMethod()
```

Comment:
We parse the encoded payment method into a `PaymentMethod` object once we’re back in `payments-core`.